modell/src/main/kotlin/no/nav/tilbakekreving/hendelse/KravgrunnlagHendelse.kt

```diff
@@ -18,16 +19,20 @@ class KravgrunnlagHendelse(
     private val ansvarligEnhet: String,
     private val kontrollfelt: String,
     // Brukes som eksternId i henting av fagsysteminfo, hva betyr det egentlig?
-    private val referanse: String,
+    val referanse: String,
     private val kravgrunnlagId: String,
-    private val perioder: List<Periode>,
+    val perioder: List<Periode>,
 ) : Historikk.HistorikkInnslag<UUID> {
     fun totaltBeløpFor(periode: Datoperiode): BigDecimal =
         perioder.single { kgPeriode -> kgPeriode.inneholder(periode) }
             .totaltBeløp()
 
     fun datoperioder() = perioder.map { it.periode }
 
+    fun totalFeilutbetalBeløpForAllePerioder() = perioder.sumOf { it.totaltBeløp() }
+
+    fun totaltFeilutbetaltPeriode() = datoperioder().minOf { it.fom } til datoperioder().maxOf { it.tom }
```

Comment:
Litt forvirret over hva denne henter men basert på koden kan dette stemme?
```suggestion
    fun feilutbetaltBeløpForPeriode() = datoperioder().minOf { it.fom } til datoperioder().maxOf { it.tom }
```

Reply:
Her hentes den totale perioden. Det vil si hvis det er flere perioder som tar vi fom fra første perioden og tom fra siste perioden. Det har ikke noe med beløp å gjøre.